P0009/P0009.md

```diff
@@ -2559,19 +2556,19 @@ template<class OtherElementType, class OtherExtents, class OtherLayoutPolicy, cl
 
     + [13.2]{.pnum} `is_assignable_v<Accessor, OtherAccessor>` is `true`;
 
-    + [13.3]{.pnum} `is_assignable_v<pointer, OtherAccessor::pointer>` is `true`;
-
-    + [13.4]{.pnum} `OtherExtents::rank() == rank()` is `true`; and
+    + [13.3]{.pnum} `OtherExtents::rank() == rank()` is `true`; and
 
-    + [13.5]{.pnum} For all `r` in the range `[0, rank())`, 
+    + [13.4]{.pnum} For all `r` in the range `[0, rank())`, 
        if `other.static_extent(r) != dynamic_extent && static_extent(r) != dynamic_extent` is `true`,
        then `other.static_extent(r) == static_extent(r)` is `true`.
 
-* [14]{.pnum} *Preconditions:* For all `r` in the range `[0, rank())`,
+* [14]{.pnum} *Mandates:* `is_assignable_v<pointer, OtherAccessor::pointer>` is `true`.
```

Comment:
I do prefer this change -- it's nicer to have the check here, rather than in the depths of the accessor.  Thanks! : - )